ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -323,6 +325,8 @@ l2leaf:
           mac_address: '0c:1d:c0:1d:62:01'
           uplink_switch_interfaces: [ Ethernet7, Ethernet7 ]
           uplink_mtu: 1498
+          ptp:
```

Comment:
I am confused. Why do you need ptp here?

Reply:
needed for some coverage in the port-channel.py. reivewed during call